bokehjs/src/coffee/models/plots/plot_canvas.coffee

```diff
@@ -178,24 +178,42 @@ export class PlotCanvasView extends BokehView
     # Update any DataRange1ds here
     frame = @model.frame
     bounds = {}
+    log_bounds = {}
+
+    calculate_log_bounds = false
+    for r in _.values(frame.x_ranges).concat(_.values(frame.y_ranges))
+      if r instanceof DataRange1d
+        if r._mapper_type == "log"
```

Comment:
Here you use a private property in an unrelated class. Either there is a public property that should be preferred or `_mapper_type` should be made public. Using `_` for indicating private members is just a suggestion and we don't enforce it too much. However, having a switch to TypeScript on the horizon, there will be no mercy for this kind of code in the future.